assets/js/pjaxinit.js

```diff
@@ -47,4 +83,13 @@
   });
   document.addEventListener('pjax:send', topbar.show);
   document.addEventListener('pjax:complete', topbar.hide);
+
+  // Add Pjax to content inserted by HTMX
+  document.addEventListener('htmx:afterSwap', function (htmxEvent) {
```

Comment:
Why are we adding HTMX related code in this pull request?

Reply:
We were improving the _scroll handling_ of Pjax, so only scroll related changes should be part of this pull requests.